src/main/kotlin/org/digma/intellij/plugin/ui/list/insights/InsightsListCellRenderer.kt

```diff
@@ -56,30 +58,39 @@ class InsightsListCellRenderer : AbstractPanelListCellRenderer() {
-            is SpanScalingRootCauseInsight -> spanScalingRootCauseItemsPanel(project, value.modelObject as SpanScalingRootCauseInsight, value.moreData)
-            is UnmappedInsight -> unmappedInsightPanel(project, value.modelObject as UnmappedInsight)
-            is NoDataYet -> noDataYetInsightPanel()
-            is NoObservability -> noObservabilityInsightPanel()
-            else -> genericPanelForSingleInsight(project, value.modelObject)
+        try {
+            val panel = when (value.modelObject) {
+                is InsightsList.GroupTitleModel -> buildGroupTitle(value.modelObject as InsightsList.GroupTitleModel)
+                is HotspotInsight -> hotspotPanel(project, value.modelObject as HotspotInsight)
+                is ErrorInsight -> errorsPanel(project, value.modelObject as ErrorInsight)
+                is LowUsageInsight -> lowUsageInsightPanel(project, value.modelObject as LowUsageInsight)
+                is NormalUsageInsight -> normalUsageInsightPanel(project, value.modelObject as NormalUsageInsight)
+                is HighUsageInsight -> highUsageInsightPanel(project, value.modelObject as HighUsageInsight)
+                is EPNPlusSpansInsight -> ePNPlusSpansPanel(project, value.modelObject as EPNPlusSpansInsight, value.moreData)
+                is SpanNPlusOneInsight -> spanNPlusOneInsightPanel(project, value.modelObject as SpanNPlusOneInsight)
+                is SlowEndpointInsight -> slowEndpointPanel(project, value.modelObject as SlowEndpointInsight)
+                is SlowestSpansInsight -> slowestSpansPanel(project, value.modelObject as SlowestSpansInsight, value.moreData)
+                is SpanUsagesInsight -> spanUsagesPanel(project, value.modelObject as SpanUsagesInsight, value.moreData)
+                is SpanDurationsInsight -> spanDurationPanel(project, value.modelObject as SpanDurationsInsight, panelsLayoutHelper)
+                is SpanDurationBreakdownInsight -> spanDurationBreakdownPanel(project, value.modelObject as SpanDurationBreakdownInsight, value.moreData)
+                is SpanSlowEndpointsInsight -> spanSlowEndpointsPanel(project, value.modelObject as SpanSlowEndpointsInsight, value.moreData)
+                is SpanScalingInsight -> spanScalingListViewItemsPanel(project, value.modelObject as SpanScalingInsight, value.moreData)
+                is SpanScalingRootCauseInsight -> spanScalingRootCauseItemsPanel(project, value.modelObject as SpanScalingRootCauseInsight, value.moreData)
+                is UnmappedInsight -> unmappedInsightPanel(project, value.modelObject as UnmappedInsight)
+                is NoDataYet -> noDataYetInsightPanel()
+                is NoObservability -> noObservabilityInsightPanel()
+                else -> genericPanelForSingleInsight(project, value.modelObject)
+            }
+
+            return panel
+        }
+        catch (ex: Exception) {
```

Comment:
handle this exception in other place:
PanelList. rebuild()

Reply:
Done